GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -357,19 +361,26 @@ protected override void OnHandleCreated(EventArgs e)
         [DllImport("user32.dll")]
         private static extern IntPtr SendMessage(IntPtr hwnd, int msg, IntPtr wp, IntPtr lp);
 
-        protected override void OnActivated(EventArgs e)
+        protected override void WndProc(ref Message m)
```

Comment:
The changes done by RussKie pushes the generic changes to the base class instead. I am dropping the suggestion for now

Reply:
> A similar change should be done in FormBrowse too.

Could you please elaborate?
Do we need to do it as part of this PR?

Reply:
Be careful on the FormBrowse not refreshing when we just switch windows in GE vs switch from notepad.

Reply:
Should we automatically refresh when we switch between windows?

Reply:
This PR deals with the modal dialog FormCommit.
There is no other window of this application which could change the repo.
For FormBrowse this can be completely different.